static/sass/_smd.scss

```diff
@@ -0,0 +1,37 @@
+.grid_separator--xl {
+  margin-bottom: $size-xxl;
+}
+
+.smd_buckets {
+  @extend %section-padding;
+  border-bottom: 1px solid $color-gray-light;
+}
+
+.bucket_card {
+  border: 1px solid $color-blue-light;
+  padding: $size-xl $size-l $size-l;
+  position: relative;
+
+  @include mq($until: bp-s) {
+    &:first-child {
+      margin-bottom: $size-l;
+    }
+  }
+
+  &--amount {
+    @extend .smallcaps;
+    font-size: $size-s;
```

Comment:
for consistency, we tend to alphabetize styles. you can quickly resort them by highlighting the group and hitting F5